mlflow/server/js/src/experiment-tracking/components/artifact-view-components/ShowArtifactLoggedModelView.tsx

```diff
@@ -113,7 +113,7 @@ class ShowArtifactLoggedModelView extends Component<Props, State> {
       // eslint-disable-next-line max-len
       `# Load model as a Spark UDF. Override result_type if the model does not return double values.\n` +
       // eslint-disable-next-line max-len
-      `loaded_model = mlflow.pyfunc.spark_udf(spark, model_uri=logged_model, result_type='double')\n\n` +
+      `loaded_model = mlflow.pyfunc.spark_udf(spark, model_uri=logged_model)\n\n` +
```

Comment:
what happens if the model doesn't have an output signature?

Reply:
It uses `double` as default result_type and shows a warning. I think this is good enough. :)